### PR TITLE
feat: Changes to Individual Learner dashboard

### PR DESCRIPTION
**Pull request by saraburns1**

### CODE REVIEW

File: tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/Learner_Summary_6da509.yaml [hunk 1]
```diff
@@ -54,9 +54,7 @@ params:
   - expressionType: SQL
     label: Grade Range
     sqlExpression: grade_bucket
-  - expressionType: SQL
-    label: Course Grade %
-    sqlExpression: ROUND(course_grade*100, 2)
```
Comment (saraburns1) [inline tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/Learner_Summary_6da509.yaml:59]: Should this have been removed? Course grade is showing like `0.3191489361702128` now

File: tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/datasets/int_problem_results.yaml [hunk 2]
```diff
@@ -199,20 +208,19 @@ columns:
   type: String
   verbose_name: Responses
 - advanced_data_type: null
-  column_name: graded
+  column_name: username
```
Comment (saraburns1) [inline tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/datasets/int_problem_results.yaml:211]: This causes the 'Problem Results' chart to fail if you select a specific student

image

File: tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/datasets/fact_video_watches.yaml [hunk 3]
```diff
@@ -146,6 +146,42 @@ columns:
   python_date_format: null
   type: String
   verbose_name: Organization
+- advanced_data_type: null
+  column_name: username
```
Comment (saraburns1) [inline tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/datasets/fact_video_watches.yaml:150]: Same problem here - Partial and Full Video views fail when a single user is selected

File: tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/Learner_Summary_6da509.yaml (hunk 1 above)
Comment (bmtcril) [reply tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/Learner_Summary_6da509.yaml:59]: This should not have been removed, it's showing the same behavior on their sandbox so you should just revert that change here (and make sure it doesn't break anything locally 😄 )

File: tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/datasets/int_problem_results.yaml (hunk 2 above)
Comment (bmtcril) [reply tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/datasets/int_problem_results.yaml:211]: We've figured out that this is due to changes in int_problems_results SQL in Superset that aren't carried back to the .sql file here automatically when they change.

**Pull request closed without merging**